src/llama-model.cpp

```diff
@@ -9777,7 +9921,7 @@ struct llm_build_mamba : public llm_graph_context {
                 cur = build_mamba_layer(rs_inp, gf, cur, model, ubatch, il);
             }
 
-            if (il == n_layer - 1 && inp_out_ids) {
+            if (il == n_layer - 1) {
```

Comment:
```suggestion
            if (il == n_layer - 1 && inp_out_ids) {
```

Reply:
Done (will push a batch of updates once CI finishes)